src/main/java/stream/flarebot/flarebot/Events.java

```diff
@@ -4,6 +4,10 @@
 import net.dv8tion.jda.core.JDA;
 import net.dv8tion.jda.core.OnlineStatus;
 import net.dv8tion.jda.core.Permission;
+import net.dv8tion.jda.core.audit.AuditLogChange;
```

Comment:
Put all the modlog stuff in its own event, much easier for handling and fixing stuff. Make another class and put everything for modlog in there instead.